pyrax/cloudcdn.py

```diff
@@ -57,14 +57,15 @@ def delete_assets(self, url=None, all=False):
 class CloudCDNServiceManager(BaseManager):
 
     def create(self, name, flavor_id, domains, origins,
-               restrictions=None, caching=None):
+               restrictions=None, caching=None, log_delivery=None):
```

Comment:
Thanks Brian for reviewing. Currently allowed value is {"enabled": "true/false"}, but in future if it supports some other key/values in this dictionary, for example {"enabled": "true", "log_path": "cloud-files-container-path"}, then we have to revisit this change.
So I am thinking current change is OK. Let me know what you think!